contrib/generic_proxy/filters/network/source/interface/filter.h

```diff
@@ -15,6 +15,20 @@ namespace GenericProxy {
 
 using ResponseUpdateFunction = std::function<void(Response&)>;
 
+/**
+ * StreamFrameHandler to handle the frames from the stream (if exists).
+ */
+class StreamFrameHandler {
+public:
+  virtual ~StreamFrameHandler() = default;
+
+  /**
+   * Handle the frame from the stream.
+   * @param frame frame from the stream.
+   */
+  virtual void onStreamFrame(StreamFramePtr frame) PURE;
```

Comment:
Is it also part of filter interface? like if the filter care about the frame, then it can implement this? 

But the request frame won't trigger this method, only the other frame will trigger this.

Reply:
This is only a interface that implemented by the terminal filter to accept the variable length request frames.
These additional frames will not be handled by the filter chain and will be handled by the terminal filter directly to avoid the complexity of full filter chain support.
This will be removed in the future, if we implemented full filter chain support.